### DISCUSSION TITLE
How can I block that content-loading placeholder on YouTube?

### DISCUSSION THREAD

**Discussion by Aleksandra0056** [lightbeam24/CustomTube#143] [Q&A]
How can I hide that content-loading placeholder skeleton which shows every time I reload YouTube?

I want to know how can I block it with a uBlock Origin content blocker.

#### I mean this:
![YouTube Content-loading placeholder](https://github.com/lightbeam24/CustomTube/assets/69351811/87b3c52f-b722-4b33-8a6c-bdf72cf9f9f7)

**Comment by lightbeam24** [comment]
I don't know how uBlock Origin does this stuff off the top of my head, so here is how you would do it in CSS:

```
.skeleton,
.watch-skeleton,
#ghost-cards {
	display: none;
}
```
